equinix/provider.go

```diff
@@ -136,6 +142,8 @@ func Provider() *schema.Provider {
 			"equinix_ecx_l2_connection":          resourceECXL2Connection(),
 			"equinix_ecx_l2_connection_accepter": resourceECXL2ConnectionAccepter(),
 			"equinix_ecx_l2_serviceprofile":      resourceECXL2ServiceProfile(),
+			"equinix_fabric_connection":          resourceFabricConnection(),
+			"equinix_fabric_service_profile":     resourceFabricServiceProfile(),
```

Comment:
It looks like `equinix_fabric_port` is missing from the resources map

Reply:
Nevermind, I was thrown off by the existence of a `resource_fabric_port.go` file, but that file seems to only have read functions.

Nit:  if we can, I think it would be preferable to put those in the data source file & rename `resource_fabric_port_acc_test.go` to `data_source_fabric_port_acc_test.go`.